### PR TITLE
feat: 9.10 for bullseye

### PR DESCRIPTION
**Pull request by develop7**

### CODE REVIEW

File: 9.10/bullseye/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    \
+    cabal --version
+
+ARG GHC=9.10.1
+ARG GHC_RELEASE_KEY=FFEB7CE81E16A36B3E2DED6F2DE04D4E97DB64AD
+
+RUN set -eux; \
+    cd /tmp; \
+    ARCH="$(dpkg-architecture --query DEB_BUILD_GNU_CPU)"; \
+    GHC_URL="https://downloads.haskell.org/~ghc/$GHC/ghc-$GHC-$ARCH-deb10-linux.tar.xz"; \
```
Comment (GrahamCampbell) [inline 9.10/bullseye/Dockerfile:89]: deb11 is needed

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]
Probably we should do bullseye-slim, too?

File: .github/workflows/debian.yml [hunk 2]
```diff
@@ -37,6 +37,10 @@ jobs:
             ghc_minor: '9.8'
           - ghc: '9.10.1'
             ghc_minor: '9.10'
+          - ghc: '9.10.1'
+            ghc_minor: '9.10'
+            deb: 'bullseye'
```
Comment (GrahamCampbell) [inline .github/workflows/debian.yml:42]: slim job missing from here, too

Comment (GrahamCampbell) [reply .github/workflows/debian.yml:42]: we need two entries

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: .github/workflows/debian.yml (hunk 2 above)
Comment (develop7) [reply .github/workflows/debian.yml:33]: I think there is:

```yaml
          - ghc: '9.10.1'
            ghc_minor: '9.10'
            deb: 'slim-bullseye'
          - ghc: '9.10.1'
            ghc_minor: '9.10'
            deb: 'bullseye'
```

**Comment by develop7** [comment]
Okay, I'll bite and merge, let's see how this works

**Pull request merged**